### ISSUE TITLE
Training data registration issue:

### ISSUE THREAD

**Issue by freblisna** [ClearAnatomics/ClearMap#81]
I've been analyzing the 2016 renier haloperidol-saline training data and I've encountered an issue in registering my detected cell coordinates to the transformed coordinates. In the saline group (ms 1272-1274), I'm seeing a failure of the registration to work effectively using the default settings, while the haloperidol works fine.  This isn't noticeable until the end when I try to align the detected cell coordinates to the registered sample and view the results via the p3d.plot viewer. When I view through the viewer, the result looks like nonsense and doesn't resemble the brain at all. 

I've inspected the alignment in Fiji as in the documentation and I don't see any obvious misalignments. The windows appear synchronized and the cursor appears to select the same landmarks between windows. When I look at my exported coordinates table for the haloperidol group, I can observe strong correlations between the non-transformed coordinate and the transformed coordinate (X vs Xt for example) where R = 0.969 to 0.999 but the saline group shows R = -0.02, or similar. Interestingly in the Saline group, one of the axes seems to show R = 1.0000 where there is no transformation at all. The orientation in the image seems to be the same for all saline and all haloperidol, and I'm using a 1,2,3 orientation tuple for all samples. 

In summary, the pipeline seems to work perfectly for haloperidol group using default settings but for some reason I'm having this issue with saline group only. 

Any thoughts on what could be happening?

**Comment by freblisna** [comment]
This was an issue unrelated to ClearMap2. I'm passing a custom cells file generated originally from cell counts in Ilastik and there was a formatting problem with this file.
